tests/unit/test_registry.py

```diff
@@ -561,3 +564,102 @@ def prepare_hooks():
         self.assertEqual([item for (item,) in sequence], [
             'wrapped',
         ])
+
+
+class ExtendedStepTest(unittest.TestCase):
+    """
+    Test extended step.
+    """
+
+    def setUp(self):
+        self.extended_step = ExtendedStep()
+        self.single_expression_placeholders = \
+            self.extended_step.single_expression_placeholders
+        self.multi_expression_placeholders = \
+            self.extended_step.multi_expression_placeholders
+
+        self.extended_step.register_placeholder('TEST1', 't1')
+        self.extended_step.register_placeholder('TEST2', 't2_1', 't2_2')
+
+    def test_register_placeholder(self):
+        """Test registering placeholders."""
```

Comment:
Does not test anything useful. How can I actually use those `TEST` placeholders?